backend/api/Controllers/RobotController.cs

```diff
@@ -643,30 +643,30 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
     /// </remarks>
     [HttpPost]
     [Authorize(Roles = Role.User)]
-    [Route("{robotId}/start-localization")]
+    [Route("start-localization")]
     [ProducesResponseType(typeof(Mission), StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public async Task<ActionResult<Mission>> StartLocalizationMission(
-        [FromRoute] string robotId,
-        [FromBody] Pose localizationPose
+        [FromBody] ScheduleLocalizationMissionQuery scheduleLocalizationMissionQuery
+
     )
     {
-        var robot = await _robotService.ReadById(robotId);
+        var robot = await _robotService.ReadById(scheduleLocalizationMissionQuery.RobotId);
         if (robot == null)
         {
-            _logger.LogWarning("Could not find robot with id={id}", robotId);
+            _logger.LogWarning("Could not find robot with id={id}", scheduleLocalizationMissionQuery.RobotId);
```

Comment:
Does this actually log the id? is id a variable here?

Reply:
Yes RobotId is a variable in scheduleLocalizationMissionQuery